x-pack/filebeat/processors/decode_cef/cef/types.go

```diff
@@ -0,0 +1,141 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cef
+
+import (
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
```

Comment:
Nit: This should go after github.com/pkg/errors

Reply:
Fixed.